### PR TITLE
Prf94 production

### PR DESCRIPTION
**Pull request by at7**
changing up the logic of how human population dumps are generated. Human data is dumped by chromosome now. I needed to updated the generation of HapMap and ESP data dumps to join up variation information and frequencies. HapMap and ESP dumps are joined and distributed as one file per population. This won't be needed anymore if we move to only reading frequency data from VCF (next release/95?).

Also a bug fix in species naming in ftp url which was reported by Ben.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm [hunk 1]
```diff
@@ -32,14 +32,15 @@ package Bio::EnsEMBL::Variation::Pipeline::ReleaseDataDumps::FinishDumpPopulatio
...
+use Compress::Zlib;
 
 sub run {
     my $self = shift;
     my $file_type = $self->param('file_type');
     if ($file_type eq 'gvf') {
         # remove allele_string and variation_id attributes from file    
         # gzip file
-        $self->clean_up_gvf_file;
+#        $self->clean_up_gvf_file;
```
Comment (sarahhunt) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm]: Is this meant to be hashed out?

Comment (at7) [reply modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm]: It was taking too long and I run the cleanup separately. It has now been added as a semaphore to the pipeline.

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm [hunk 2]
```diff
@@ -51,64 +53,64 @@ sub clean_up_vcf_files {
   my $self = shift;
   my $pipeline_dir = $self->param('pipeline_dir');
   my $tmp_dir = $self->param('tmp_dir');
-  my $human_vcf_dir = "$pipeline_dir/vcf/Homo_sapiens/";
+  my $human_vcf_dir = "$pipeline_dir/vcf/homo_sapiens/";
   die "$human_vcf_dir is not a directory" unless (-d $human_vcf_dir);
 
-  opendir(my $dh, "$pipeline_dir/vcf/Homo_sapiens") or die $!;
+  opendir(my $dh, "$pipeline_dir/vcf/homo_sapiens") or die $!;
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm]: Could use the variable `$human_vcf_dir` instead

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm [hunk 3]
```diff
@@ -51,64 +53,64 @@ sub clean_up_vcf_files {
...
   my @dir_content = readdir($dh);
   closedir($dh);
   foreach my $file (@dir_content) {
     if ($file =~ m/\.vcf$/) {
-      my $vcf_file = "$pipeline_dir/vcf/Homo_sapiens/$file";
+      my $vcf_file = "$pipeline_dir/vcf/homo_sapiens/$file";
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm]: Could replace it by `my $vcf_file = "$human_vcf_dir/$file";`

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm [hunk 4]
```diff
@@ -51,64 +53,64 @@ sub clean_up_vcf_files {
...
       $self->run_cmd("vcf-sort < $vcf_file | bgzip > $vcf_file.gz");
       $self->run_cmd("rm $vcf_file");
     }
     if ($file =~ m/\.out$/ || $file =~ m/\.err$/) {
-      $self->run_cmd("rm $pipeline_dir/vcf/Homo_sapiens/$file");
+      $self->run_cmd("rm $pipeline_dir/vcf/homo_sapiens/$file");
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm]: Could replace it by `$self->run_cmd("rm $human_vcf_dir/$file");`

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/InitDumpPopulation.pm [hunk 5]
```diff
@@ -71,29 +71,17 @@ sub fetch_input {
...
-    my $human_gvf_file = $self->param('pipeline_dir') . '/gvf/Homo_sapiens/Homo_sapiens.gvf';
-    system("gunzip $human_gvf_file_gz");
-
-    $self->param('human_gvf_file', $human_gvf_file);
-    $self->param('gvf_files_dir', "$pipeline_dir/gvf/Homo_sapiens/");
   }
 
   if ($file_type eq 'vcf') {
-    $self->param('vcf_files_dir', "$pipeline_dir/vcf/Homo_sapiens/");
+    $self->param('vcf_files_dir', "$pipeline_dir/vcf/homo_sapiens/");
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/InitDumpPopulation.pm]: Actually wondering if `$pipeline_dir/vcf/homo_sapiens/` should be setup/declared as a pipeline variable

**Pull request merged**